File: HA5_1.cpp
//Write a program which accept number from user and print that number of $&* on screen 
//Input : 5
//Output : $&* $&* $&* $&* $&*

#include<iostream>
using namespace std;

void Pattern(int iNo)
{
    int iCnt = 0;
    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        cout<<"$   &   *"<<"\t";
    }
}

int main()
{
    int iValue = 0;
    cout<<"Enter a Number"<<"\n";
    cin>>iValue;

    Pattern(iValue);

    return 0;
}

File: HA5_1.py
#Write a program which accept number from user and print that number of $&* on screen 
#Input : 5
#Output : $&* $&* $&* $&* $&*

def Pattern(iNo):
    iCnt = 0
    for iCnt in range(0, iNo,1):
        print("$    &   *\t")

def main():
    print("Enter a Number")
    iValue = int(input())

    Pattern(iValue)

if __name__ =="__main__":
    main()